justfile

```diff
@@ -25,7 +25,7 @@ default:
 # Install necessary tools
 install-tools:
     rustup component add rustfmt --toolchain nightly
-    cargo install cargo-sort cargo-nextest
+    cargo install taplo-cli cargo-nextest
```

Comment:
The following recipes are still using `cargo sort` command: `check-tools`, `format` and `format-check`.They must be replaced with `taplo fmt --check` instead, right?